js/components/NoteList/AddNote/styles.js

```diff
@@ -8,7 +8,7 @@ const styles = StyleSheet.create({
     marginTop: 22,
   },
   container: {
-    top: height / 3,
+    top: height / 3.7,
```

Comment:
do not use height and width for margin, padding or top/bottom/left/right. Thank you